sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRepository.java

```diff
@@ -0,0 +1,247 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.containers.containerregistry;
+
+import com.azure.containers.containerregistry.models.ArtifactManifestProperties;
+import com.azure.containers.containerregistry.models.ContentProperties;
+import com.azure.containers.containerregistry.models.DeleteRepositoryResult;
+import com.azure.containers.containerregistry.models.ManifestOrderBy;
+import com.azure.containers.containerregistry.models.RepositoryProperties;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.exception.ClientAuthenticationException;
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+
+/**
+ * This class provides a client that contains all the operations for repositories in Azure Container Registry.
+ * Operations allowed by the client are listing, retrieving, deleting, setting writeable properties.
+ * These operations are supported on the repository and the respective tags and manifests in it.
+ *
+ * <p><strong>Instantiating Container Repository Client</strong></p>
+ *
+ * {@codesnippet com.azure.containers.containerregistry.repository.instantiation}
+ *
+ * <p>View {@link ContainerRegistryClientBuilder this} for additional ways to construct the client.</p>
+ *
+ * @see ContainerRegistryClientBuilder
+ */
+public final class ContainerRepository {
```

Comment:
We took away 'Client' from these types because we have reduced ourselves to only one builder. @Srikanta did mention that we can choose to append Client to these types even though we have one Builder since they still call servicemethods - I will follow up with Jonathan on this one and either change the fieldName to typeName.